src/actions/index.js

```diff
@@ -56,3 +57,10 @@ export const actionsLogout = () => async dispatch => {
       console.log(err);
     });
 };
+
+export const actionsToggle = link => dispatch => {
+  return dispatch({
+    type: TOGGLE,
+    payload: link
+  });
+};
```

Comment:
I don't see this chain of functions being used anywhere and the component works fine, so I think it can be removed.

Reply:
removed